feat: Add Installation Guide for @sentry/capacitor (#3730)
getsentry/sentry-docs: src/includes/getting-started-install/javascript.capacitor.mdx

> To install the Sentry Capacitor SDK, use npm or yarn to install it alongside the corresponding Sentry SDK for the framework you're using such as Angular.

Reviewer: ```suggestion
Install the Sentry Capacitor SDK alongside the corresponding Sentry SDK for the framework you're using, such as Angular in this example:
```
